pkgs/data/themes/everforest-gtk-theme/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  pname = "Everforest-GTK-Theme";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/Fausto-Korpsvart/Everforest-GTK-Theme/archive/refs/heads/master.tar.gz";
+    sha256 = "2gmSMW/yUdinvZ/Jaw6GyfEL/CBqYKtw0QrYWy9KrL0=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/themes
+    rm -rf README.md LICENSE icons extra
+    cp -r themes/* $out/share/themes/
+  '';
+
+  meta = with lib; {
+    description = "Everforest theme for GTK based desktop environments";
+    homepage = "https://github.com/Fausto-Korpsvart/Everforest-GTK-Theme";
+    license = licenses.gpl3Only;
+    platforms = platforms.unix;
```

Comment:
```suggestion
    platforms = platforms.all;
```

Reply:
Comparing to catppuccin-gtk, https://github.com/NixOS/nixpkgs/blob/980433df445ceadad8c446267dc5a9d029055239/pkgs/data/themes/catppuccin-gtk/default.nix#L89, should this not be `platforms.linux`.

Reply:
That is definitely wrong, gtk themes can be used on macOS and other unix-like distributions, and even kinda work on Windows.